test/unit/account-link-test.js

```diff
@@ -3,8 +3,8 @@ var linkGen = require('../../ui/lib/account-link')
 
 describe('account-link', function() {
 
-  it('adds testnet prefix to morden test network', function() {
-    var result = linkGen('account', '2')
+  it('adds testnet prefix to ropsten test network', function() {
+    var result = linkGen('account', '3')
     assert.notEqual(result.indexOf('testnet'), -1, 'testnet injected')
     assert.notEqual(result.indexOf('account'), -1, 'account included')
```

Comment:
why do we have the different language here? injected vs included -- i know its not part of the diff...